controllers/stickers/uploadStickers.js

```diff
@@ -21,23 +21,21 @@ exports.createSticker = async (body) => {
     jerseyNumber
   });
 
-  return newSticker;
+  return responses.singleDTOResponse(res,200,"Se creo con exito el nuevo sticker",newSticker);
 }
 
 //funcion de subir imagenes de los cromos
 exports.uploadFileSticker = async (req, res) => {
     if (!req.file?.path) {
-        return res.status(400).json({
-            success: false,
-            message: "No se ha subido archivo o no cumple el filtro",
-        })
+        return responses.errorDTOResponse(res,400,"No se ha subido archivo o no cumple el filtro");
     }
 
     try {
       const newSticker = await createSticker({
         ...req.body,
         fileName: req.file.filename
       });
+      responses.single
       res.status(201).json({
         message: 'Cromo creado con exito',
         item: newSticker
```

Comment:
```suggestion
      return responses.singleDTOResponse(res, 201, "Cromo creado con exito", newSticker);
```
Mejor esto, lo anterior no compila